frontend/src/App.js

```diff
@@ -80,12 +75,29 @@ class App extends Component {
 
   render() {
     return (
-      <div style={{background: "#f7f7f7"}} className="App">
-      <header>
-        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous" />
-      </header>
-        <NAVBAR handler={this.updateInfo} />
-
+      <div className="App">
+        <header>
+          <link
```

Comment:
external stylesheets should be declared in document head or imported as npm module. (document head is in `public/index.html`)